### ISSUE TITLE
Vowel & Speech model: LPG-Harmonics knob incorrectly mimicks LPG-Morph and LPG-Frequency

### ISSUE THREAD

**Issue by xandramax** [Xenakios/Atelier#3]
Environment: Windows 7

The LPG (Decay Envelope) -> Harmonics attenuverter works on all models except this one, as evidenced by both the audio and the visual feedback.

**Comment by xandramax** [comment]
Apparently this LPG modulation actually _does_ function, but only when the Harmonics (MODE) knob is set within the first 50% of the Formant (FOR.) region.

Starting from fully clockwise, the LPG modulation progressively attenuates as the knob is turned clockwise and reaches zero (no modulation) at 50% through the Formant region.

**Comment by Xenakios** [comment]
The Vowel&Speech model certainly seems to be the odd one out in how it functions...I can take a look if the behavior can be improved, but if it requires changing too much code, I probably won't fix it.

**Comment by xandramax** [comment]
I'm going to try and get to the bottom of this one. The behavior seems to be more complex than I realized at first; after playing with this more it seems that this is actually related to the (perfectly normal) behavior described issue #2. There are two behaviors driving what I've described in this issue:
- Starting at fully counterclockwise, as the Harmonics knob is rotated toward 11 o'clock, the LPG-Morph and LPG-Frequency knobs have their effect attenuated to zero before switching function to speed and intonation. However, unexpectedly, the LPG-Harmonics knob also has its effect attenuated to zero despite there being no related change in function when Harmonics is past 11 o'clock. In contrast, the LPG-Timbre knob always functions as normal.
- Non-zero voltage present at the Harmonics-CV input affects this behavior _even when its attenuverter is at 12 o'clock_, as if the Harmonics knob is being offset, even though there is not an actual change in the Harmonics (neither in terms of audio nor visual feedback). This "ghost offset" is responsible for my measurement of "50% through the Formant region" mentioned in my comment reply above.

So, in fewer words:
- The new LPG-Harmonics knob seems to be accurately mimicking LPG-Morph & LPG-Frequency, when it should instead be mimicking LPG-Timbre. 
- Non-zero voltage at the Harmonics-CV input has some effect even when its new attenuverter is fully closed.

I've separated the latter point as issue #5.
